src/invidious/yt_backend/extractors.cr

```diff
@@ -135,6 +135,10 @@ private module Parsers
         when "Premium"
           # TODO: Potentially available as item_contents["topStandaloneBadge"]["metadataBadgeRenderer"]
           badges |= VideoBadges::Premium
+        when "Members only"
```

Comment:
There is also `Members first` (from https://github.com/FreeTubeApp/FreeTube/pull/7208#issuecomment-2816593838
![image](https://github.com/user-attachments/assets/6d258d63-c4b7-463b-a06f-9dc88e477378)

![image](https://github.com/user-attachments/assets/a98c527e-7bc1-47e0-bb65-969423d18c1b)